### DISCUSSION TITLE
Artisan command to do a lint check for dump calls

### DISCUSSION THREAD

**Discussion by adrianspacely** [spatie/laravel-ray#66] [Ideas]
Would be neat if there was an artisan command to do a lint check for dump calls. I.e. `artisan ray:lint`.

This would be great for CI and peace of mind that stray `dd`, `dump` `var_dump`, `ray`, etc. calls aren't in the codebase after debugging.

**Comment by freekmurze** [comment]
I'd definitely accept a PR for that!
